backend/src/sms/middlewares/sms-callback.middleware.ts

```diff
@@ -48,7 +65,26 @@ const parseEvent = async (
   }
 }
 
+const parseTransactionalEvent = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+): Promise<Response | void> => {
+  try {
+    await SmsCallbackService.parseTransactionalEvent(req)
+    logger.info({
+      message: 'Successfully parsed transactional event',
+      action: 'parseTransactionalEvent',
+    })
+    return res.sendStatus(200)
+  } catch (err) {
+    return next(err)
+  }
```

Comment:
No need for this try/catch wrapping as it's already the default behavior

Reply:
done